### PR TITLE
zanshin kf5 release, bomber fix

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: kde-misc/zanshin/zanshin-0.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+EAPI=6
+
+CMAKE_MIN_VERSION="3.2.0"
+KDE_TEST="forceoptional"
+VIRTUALX_REQUIRED="test"
+inherit kde5
+
+DESCRIPTION="Getting things done application by KDE"
+HOMEPAGE="https://zanshin.kde.org/"
+[[ ${PV} == 9999 ]] || SRC_URI="http://files.kde.org/zanshin/${P}.tar.bz2"
```
Comment (johu) [inline kde-misc/zanshin/zanshin-0.4.0.ebuild]: KDE_BUILD_TYPE

File: kde-misc/zanshin/zanshin-0.4.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+LICENSE="|| ( GPL-2 GPL-3 )"
+KEYWORDS="~amd64"
+IUSE=""
+
+# FIXME: bundles libkdepim
```
Comment (johu) [inline kde-misc/zanshin/zanshin-0.4.0.ebuild]: is this still valid? do you tried to unbundle it?

File: kde-misc/zanshin/zanshin-9999.ebuild [hunk 3]
```diff
@@ -11,13 +11,14 @@ inherit kde5
 
 DESCRIPTION="Getting things done application by KDE"
 HOMEPAGE="https://zanshin.kde.org/"
+[[ ${PV} == 9999 ]] || SRC_URI="http://files.kde.org/zanshin/${P}.tar.bz2"
```
Comment (johu) [inline kde-misc/zanshin/zanshin-9999.ebuild]: same as release version comment

File: kde-misc/zanshin/zanshin-9999.ebuild [hunk 4]
```diff
@@ -11,13 +11,14 @@ inherit kde5
...
 
 LICENSE="|| ( GPL-2 GPL-3 )"
 KEYWORDS=""
 IUSE=""
 
 # FIXME: bundles libkdepim
```
Comment (johu) [inline kde-misc/zanshin/zanshin-9999.ebuild]: same as release version comment

File: kde-misc/zanshin/zanshin-0.4.0.ebuild (hunk 2 above)
Comment (a17r) [inline kde-misc/zanshin/zanshin-0.4.0.ebuild]: yes, it looks very much out of date/sync compared to libkdepim.git so didn't try

**Comment by johu** [comment]
Hint: Commit message subject for a version increase on a package.

**Comment by johu** [comment]
Thanks, picked.

**Pull request closed without merging**